.github/workflows/AutoAssignToSprint.yml

```diff
@@ -0,0 +1,18 @@
+name: Auto Assign to Sprint Project
+
+on:
+  issues:
+    types: [opened]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to One Project
+    steps:
+    - name: Assign NEW issues to sprint proect
```

Comment:
typo

Reply:
Also it'll be good to rename this to  `OnNewIssueOrPR` as that's the trigger, we can add more jobs that's irrelevant to assigning projects in the future in the same script